samples/HelloWorld/README.md

```diff
@@ -0,0 +1,58 @@
+# Salesforce App
```

Comment:
We kind of talked about this offline: do we think this README should be a level up, in the `samples/` folder? It seems like, as we potentially add more samples, the README content really applies to all of it. What are your thoughts? @rwhitley ?

Reply:
Oh wait nevermind I think I have the wrong README. 😛

Reply:
Yeah, this is the ReadMe that is automatically generated when you create a new LWC project.